src/appleseed/renderer/modeling/project/project.xsd

```diff
@@ -294,22 +294,6 @@
                         </xsd:complexContent>
                     </xsd:complexType>
                 </xsd:element>
-                <xsd:element name="rules" minOccurs="0">

```

Comment:
Not loading them is one thing, but failing to load a project file altogether because it now contains unknown elements is another...